src-mobile/script/components/checkPlaybackbarTextAndAddCarousel.js

```diff
@@ -0,0 +1,34 @@
+import { putFade } from "./putFade.js";
+
+export function checkplaybackbarText() {
+  const playbackbar = document.querySelector(`.playbackbar-track-name`);
+
+  const textElement = playbackbar.textContent;
+
+  switch (true) {
+    case textElement.length > 21 && textElement.length < 40:
```

Comment:
this function not cover if the length is equal 40

Reply:
Fixed whitin the next commit